src/cpp/benders/benders_core/BendersBase.cpp

```diff
@@ -744,10 +815,14 @@ WorkerMasterPtr BendersBase::get_master() const { return _master; }
 
 void BendersBase::addSubproblem(
     const std::pair<std::string, VariableMap> &kvp) {
-  subproblem_map[kvp.first] = std::make_shared<SubproblemWorker>(
-      kvp.second, GetSubproblemPath(kvp.first),
-      SubproblemWeight(_data.nsubproblem, kvp.first), _options.SOLVER_NAME,
-      _options.LOG_LEVEL, log_name());
+  subproblem_map[kvp.first] = makeSubproblemWorker(kvp);
+}
+std::shared_ptr<SubproblemWorker> BendersBase::makeSubproblemWorker(
+    const std::pair<std::string, VariableMap> &kvp) const {
+  return std::make_shared<SubproblemWorker>(
+        kvp.second, GetSubproblemPath(kvp.first),
+      SubproblemWeight(_data.nsubproblem, kvp.first), Options().SOLVER_NAME,
+      Options().LOG_LEVEL, log_name());
```

Comment:
Why do we use `Options()` instaed of `options_` ?

Reply:
It makes no difference, Options() is just a getter on options_